draft-ietf-tls-iana-registry-updates.md

```diff
@@ -556,6 +591,12 @@ Note:
 : Certificate Types marked as "Yes" are those allocated via Standards
 Track RFCs.  Certificate Types marked as "No" are not.
 
+Note:
+: If an item is not marked as recommended it does not necessarily mean
+that it is flawed, rather, it indicates that either the item has not
```

Comment:
I might consider a semicolon instead of a comma before "rather", but trust the RFC Editor to do the right thing.